cypress/e2e/tests/pages/explorer/api/api-services.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { APIServicesPagePo } from '@/cypress/e2e/po/pages/explorer/api-services.po';
+import SortableTablePo from '@/cypress/e2e/po/components/sortable-table.po';
+
+describe('Cluster Explorer', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  describe('API: APIServices', () => {
+    let apiServicesPage: APIServicesPagePo;
+
+    beforeEach(() => {
+      apiServicesPage = new APIServicesPagePo('local');
+      apiServicesPage.goTo();
+    });
+
+    it('Should have a title', () => {
+      cy.get('h1').should('contain', 'APIServices');
+    });
+
+    it('Should be able to use shift+j to select corre', () => {
+      cy.get('.main-layout').click();
+
+      const sortableTable = new SortableTablePo('.sortable-table');
```

Comment:
This page is a ResourceList page as mentioned [here](https://github.com/rancher/dashboard/pull/9089#discussion_r1224421773). 

The page po should return an instance of `BaseResourceList` and the sortable table is then accessed through it